models/marts/core/int_all_columns.sql

```diff
@@ -0,0 +1,59 @@
+-- one row for each column in a node or source
+
+with
+
+stg_columns as (
+
+    select
+        node_unique_id,
+        name,
+        description,
+        data_type,
+        quote
+
+    from {{ ref('stg_columns') }}
+
+),
+
+stg_nodes as (
+
+    select
+        unique_id,
+        name,
+        resource_type,
+        file_path,
+        is_enabled,
+        database,
+        schema,
+        package_name,
+        alias
+
+    from {{ ref('stg_nodes') }}
+
+),
+
+final as (
+
+    select
+        stg_nodes.unique_id as node_unique_id,
+        stg_nodes.name as node_name,
+        stg_nodes.resource_type as node_resource_type,
+        stg_nodes.file_path as node_file_path,
+        stg_nodes.is_enabled as node_is_enabled,
+        stg_nodes.database as node_database,
+        stg_nodes.schema as node_schema,
+        stg_nodes.package_name as node_package_name,
+        stg_nodes.alias as node_alias,
+        stg_columns.name as name,
+        stg_columns.description as description,
+        stg_columns.data_type as data_type,
+        stg_columns.quote as quote
+
+    from stg_columns
+
+    inner join stg_nodes
```

Comment:
`stg_nodes` does not contain sources, so this inner join is likely removing columns from sources. Is there a need to join in all this node-level info to this view? wondering if it makes more sense to just leave this fairly untransformed, and join in node-level info when needed in future fct models?

Reply:
I can leave it untransformed, with the only node-related column coming from `stg_columns`: `node_unique_id`.

I aimed to provide as much information as possible because:
1. I presumed that having One Big Table is preferable in this case to limit potential joins.
2. Besides the missing descriptions and used names, repeated names, I can't think of any other features that depend solely on pure columns' information. Perhaps more sophisticated use-cases will be in relation to a node.
3. In other `int_` models like `int_all_dag_relationships`, we do include parent's/children's data, but we could just as well say: `parent_node_id` and `child_node_id`. Then you could join twice with `int_all_graph_resources` and achieve the same outcome, but for some reason, all the data ended up in this One Big Table `int_all_dag_relationships`.

As mentioned initially, it's up to you to decide the preferred way of modeling this data :)

Reply:
hey @gastlich ! we spoke this morning, and decided that we should remove this model, and just leave column information relatively untransformed up to `stg_columns` -- then users  can build and join from there as necessary to create their own rules!

Reply:
@dave-connors-3 I've removed `int_all_columns` model. Changed the documentation and removed any references to the deleted model. Please have a look :)